adapter/README.md

```diff
@@ -100,25 +100,25 @@ captain.row();
 ## Applicability
 Use the Adapter pattern when
 
-* you want to use an existing class, and its interface does not match the one you need
-* you want to create a reusable class that cooperates with unrelated or unforeseen classes, that is, classes that don't necessarily have compatible interfaces
-* you need to use several existing subclasses, but it's impractical to adapt their interface by subclassing every one. An object adapter can adapt the interface of its parent class.
-* most of the applications using third party libraries use adapters as a middle layer between the application and the 3rd party library to decouple the application from the library. If another library has to be used only an adapter for the new library is required without having to change the application code.
+* You want to use an existing class, and its interface does not match the one you need
+* You want to create a reusable class that cooperates with unrelated or unforeseen classes, that is, classes that don't necessarily have compatible interfaces
+* You need to use several existing subclasses, but it's impractical to adapt their interface by subclassing every one. An object adapter can adapt the interface of its parent class.
+* Most of the applications using third-party libraries use adapters as a middle layer between the application and the 3rd party library to decouple the application from the library. If another library has to be used only an adapter for the new library is required without having to change the application code.
 
 ## Consequences:
 Class and object adapters have different trade-offs. A class adapter
 
-*	adapts Adaptee to Target by committing to a concrete Adaptee class. As a consequence, a class adapter won’t work when we want to adapt a class and all its subclasses.
-*	let’s Adapter override some of Adaptee’s behavior, since Adapter is a subclass of Adaptee.
-*	introduces only one object, and no additional pointer indirection is needed to get to the adaptee.
+*	Adapts Adaptee to Target by committing to a concrete Adaptee class. As a consequence, a class adapter won’t work when we want to adapt a class and all its subclasses.
```

Comment:
Will this be `adopt`?

Or should we stick to something like `change` or `modify`?

```suggestion
*	Adapts Adaptee to Target by committing to a concrete Adaptee class. As a consequence, a class adapter won’t work when we want to adopt a class and all its subclasses.
```

Reply:
@iluwatar any thoughts on this?

Reply:
> @iluwatar any thoughts on this?

My original thought was to use the verb `adapt` as in the adapter.